src/users/registration.md

```diff
@@ -61,24 +61,14 @@ Deis Workflow supports three registration modes:
 | disabled          | Does not allow anyone to register new users.    |
 | admin\_only       | Only existing admins may register new users     |
 
-To modify the registration mode for Workflow you must add or modify the `REGISTRATION_MODE` environment variable. If
-Deis Workflow is already up and running, you may use `kubectl --namespace=deis edit rc deis-controller`:
+To modify the registration mode for Workflow you must add or modify the `REGISTRATION_MODE` environment variable for the
+controller component. If Deis Workflow is already running, you may use `kubectl --namespace=deis edit deployment deis-controller`:
```

Comment:
I'll see if I can work up a `patch` invocation that works with deployments. First pass is giving me grief!